tensorflow/inference/docker/2.5/py3/Dockerfile.cpu

```diff
@@ -29,7 +29,8 @@ ENV DEBIAN_FRONTEND=noninteractive
 
 # nginx + njs
 RUN apt-get update \
- && apt-get -y upgrade \
```

Comment:
is running `apt-get -y upgrade` not updating `systemd`

Reply:
It is, but it is also upgrading many packages under the hood. We want to maintain consistency in our build system as much as possible from build to build, so we're only updating packages we need to upgrade.